content/docs/0.8.x/operate/multi_cluster/index.md

```diff
@@ -0,0 +1,220 @@
+  * Option 3: Expose Keptn via a **Ingress**
+  * Option 4: Access Keptn via a **Port-forward**
+
+    The next figure shows an overview of the four ways to expose Keptn:
+
+    {{< popup_image
+    link="./assets/installation_options.png"
+    caption="Installation options"
+    width="1000px">}}
+
+* Please make yourself familiar with the ways of exposing Keptn as explained [here](../install/#create-or-bring-a-kubernetes-cluster). Then come back and continue installing Keptn control plane.
+
+* To install the control plane, execute `keptn install` with the option you chose for exposing Keptn:
+
+    ```console
+    keptn install --endpoint-service-type=[LoadBalancer, NodePort, ClusterIP]
+    ```
+
+## Install Keptn Execution plane
+
+In this release of Keptn, the execution plane services for deployment (`helm-service`) and testing (`jmeter-service`) can be installed via Helm Charts. 
+
+Please find the Helm Charts here: 
+
+  - `helm-service`: GitHub Release for [0.8.0](https://github.com/keptn/keptn/releases/tag/0.8.0) at **Assets** > `helm-service-0.8.0-dev.tgz`
+
+  - `jmeter-service`: GitHub Release for [0.8.0](https://github.com/keptn/keptn/releases/tag/0.8.0) at **Assets** > `jmeter-service-0.8.0-dev.tgz`
+
+### How to deploy an execution plane services?
+
+* Download the `values.yaml` from the release branch, e.g., for the jmeter-service:
+
+    ```
+    wget https://raw.githubusercontent.com/keptn/keptn/release-0.8.0/jmeter-service/chart/values.yaml
+    ```
+
+* Edit the `values.yaml` to connect the services to the Keptn control plane, identified by its endpoint and API token. Therefore, set the values (1) - (5):
+
+    ```
+    remoteControlPlane:
+      enabled: true                         # < (1) set to true
+      api:
+        protocol: "http"                    # < (2) set protocol: http or https
+        hostname: ""                        # < (3) set Keptn hostname (without /api)
+        apiValidateTls: true                # < (4 - optional) option to skip TLS verification
+        token: ""                           # < (5) set Keptn API token
+    ```
+
+* Depending on your setup of the multi-cluster environment and the approach you modeled your staging process, one stage can be for example on a seperate cluster. Let's assume the following setup: 
+  
+  * Project: `sockshop`
+  * Service: `carts`
+  * Stages: 
+      * `hardening` - on Cluster-A
+      * `production` - on Cluster-B  
+
+    To properly configure the execution plane services that run, for example, on **Cluster-A**, the distributor in the `values.yaml` needs to be configured:
+
+    ```
+    distributor:
+      projectFilter: ""                     # set the project, e.g., "sockshop" (to get events for the entire project)
+      stageFilter: ""                       # set the stage, e.g., "hardening" (to get events for the stage)
+      serviceFilter: ""                     # set the service, e.g., "carts" (to get events for the service )
+    ``` 
```

Comment:
In the above example, shouldn't the configuration for **Cluster-A** have `stageFilter` set to `hardening`?

Reply:
that is true.